filebeat/module/elasticsearch/audit/ingest/pipeline-plaintext.json

```diff
@@ -11,17 +11,18 @@
                     "ES_AUDIT_EVENT_TYPE": "\\[%{WORD:event.type}\\]",
                     "ES_AUDIT_ORIGIN_TYPE": "(origin_type\\=\\[%{WORD:elasticsearch.audit.origin.type}\\])?",
                     "ES_AUDIT_ORIGIN_ADDRESS": "(origin_address\\=\\[%{IPORHOST:source.ip}\\])?",
-                    "ES_AUDIT_PRINCIPAL": "(principal\\=\\[%{WORD:user.name}\\])?",
-                    "ES_AUDIT_REALM": "(realm\\=\\[%{WORD:elasticsearch.audit.user.realm}\\])?",
+                    "ES_AUDIT_PRINCIPAL": "(principal\\=\\[%{DATA:user.name}\\])?",
+                    "ES_AUDIT_REALM": "(realm\\=\\[%{WORD:elasticsearch.audit.realm}\\])?",
```

Comment:
Breaking change or was never release?

Reply:
I think this qualifies as a breaking change. Previously we thought that the value of `realm=[...]` was the authenticated user's realm (`es.audit.user.realm`) but it turns out to be the realm that was used to test against before the user was authenticated or not (`es.audit.realm`).

The JSON audit logs contain both fields, but the plaintext audit logs only contain the latter one (`es.audit.realm`). Hence this change.

From a ES mapping perspective, we're introducing a new field here, `es.audit.realm` so in that sense it's not a breaking change.